wgengine/userspace.go

```diff
@@ -341,11 +341,13 @@ func (e *userspaceEngine) Reconfig(cfg *wgcfg.Config, routerCfg *router.Config)
 	defer e.wgLock.Unlock()
 
 	peerSet := make(map[key.Public]struct{}, len(cfg.Peers))
+	var peerKeys []string
 	e.mu.Lock()
 	e.peerSequence = e.peerSequence[:0]
 	for _, p := range cfg.Peers {
 		e.peerSequence = append(e.peerSequence, p.PublicKey)
 		peerSet[key.Public(p.PublicKey)] = struct{}{}
+		peerKeys = append(peerKeys, key.Public(p.PublicKey).ShortString())
```

Comment:
From now on, we only want to print nodes we *are* connected to in the peers: log line, not every peer in the network map. That list will change more frequently than Reconfig() (which gets a list of all nodes in the network map, whether or not we've tried to connect to them yet).

The best place to generate (and print) the peerKeys list will be at exactly the same time as you print the peers: line. As a result, you probably want to remove this entire loop from this section, and manage the peerSequence separately at the point when peers: is printed.